src/App.js

```diff
@@ -64,6 +65,21 @@ class App extends Component {
   }
 }
 
+App.propTypes = {
+  structureURL: PropTypes.string.isRequired,
+  waveformURL: PropTypes.string.isRequired,
+  audioURL: PropTypes.string.isRequired,
+  streamDuration: PropTypes.number.isRequired,
+  initStructure: PropTypes.object,
```

Comment:
Should this be required as it says in the README?
```suggestion
  initStructure: PropTypes.object.isRequired,
```

Reply:
Good catch! :100: